portal/src/main/scala/cromwell/pipeline/service/UserService.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dao.repository.UserRepository
+import cromwell.pipeline.datastorage.dto.User.UserEmail
+import cromwell.pipeline.datastorage.dto.{ UserDeactivationByEmailResponse, UserDeactivationByIdResponse, UserId }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class UserService(userRepository: UserRepository)(implicit executionContext: ExecutionContext) {
+  def deactivateByEmail(email: UserEmail): Future[Option[UserDeactivationByEmailResponse]] =
+    userRepository
+      .deactivateByEmail(email)
+      .flatMap { _ =>
+        userRepository.getUserByEmail(email)
+      }
+      .map { getUser =>
+        getUser.map { user =>
+          UserDeactivationByEmailResponse(user.email, user.active)
+        }
+      }
```

Comment:
Why do we need all of these actions?
If `deactivateByEmail` returns `Success`, we already have enough info to conclude whether the user was deactivated or not. Also we have enough information to create `UserDeactivationByEmailResponse`. However, I'm still don't understand why do we need to send back `UserDeactivationByEmailResponse`.
Same applies to `deactivateById` method.